.gitignore

```diff
@@ -17,6 +17,7 @@ dist
 .env.development.local
 .env.test.local
 .env.production.local
+clientSecret.json
```

Comment:
This is a file I keep in my local folder now - it has the client secret from google. We don't need it unless we also build a server side application, but I'm keeping it there just in case.

It's not good to expose our client secret in github, so I added it to the .gitignore